eng/testing/tests.mobile.targets

```diff
@@ -8,7 +8,7 @@
 
   <!-- Generate a self-contained app bundle for Android with tests. -->
   <UsingTask TaskName="AndroidAppBuilderTask" 
-             AssemblyFile="$(AndroidAppBuilderDir)AndroidAppBuilder.dll" />
+             AssemblyFile="$(AndroidAppBuilderTasksAssemblyPath)" />
```

Comment:
For correctness, we should condition these `UsingTask` for the target that they're only used.